axelor-account/src/main/java/com/axelor/apps/account/service/moveline/massentry/MoveLineMassEntryRecordServiceImpl.java

```diff
@@ -258,4 +264,45 @@ public MoveLineMassEntry setInputAction(MoveLineMassEntry moveLine, Move move) {
     }
     return moveLine;
   }
+
+  @Override
+  public void setAnalyticMoveLineMassEntryList(
+      MoveLineMassEntry moveLineMassEntry, MoveLine moveLine) {
+    moveLineMassEntry.clearAnalyticMoveLineMassEntryList();
+    if (CollectionUtils.isNotEmpty(moveLine.getAnalyticMoveLineList())) {
+      for (AnalyticMoveLine analyticMoveLine : moveLine.getAnalyticMoveLineList()) {
+        AnalyticMoveLine copyAnalyticMoveLine =
+            analyticMoveLineRepository.copy(analyticMoveLine, false);
+        moveLineMassEntry.addAnalyticMoveLineMassEntryListItem(copyAnalyticMoveLine);
+      }
+    }
+  }
+
+  @Override
+  public void setAnalyticMoveLineList(MoveLineMassEntry moveLineMassEntry, MoveLine moveLine) {
+    moveLine.clearAnalyticMoveLineList();
+    if (CollectionUtils.isNotEmpty(moveLineMassEntry.getAnalyticMoveLineMassEntryList())) {
+      for (AnalyticMoveLine analyticMoveLine :
+          moveLineMassEntry.getAnalyticMoveLineMassEntryList()) {
+        AnalyticMoveLine copyAnalyticMoveLine =
+            analyticMoveLineRepository.copy(analyticMoveLine, false);
+        moveLine.addAnalyticMoveLineListItem(copyAnalyticMoveLine);
+      }
+    }
+  }
+
+  @Override
+  public void setAnalyticMoveLineMassEntryList(MoveLineMassEntry moveLine) {
```

Comment:
Same as before.
Also please rename parameter to `moveLineMassEntry`

Reply:
Resolved